### ISSUE TITLE
Add an `ignorePaths` option to the global middleware

### ISSUE THREAD

**Issue by BracketJohn** [sidebase/nuxt-auth#91]
### Describe the feature

Add an `ignorePaths` option to the global middleware, so that pages can be excluded from the global protetction without adding:
```ts
definePageMeta({ auth: false }) 
```
to them.

This approach has some advantages:
- it works if a project hast just a `app.vue`. Currently, in this case, the `app.vue` cannot be excluded as the `definePageMeta` macro does not work on it
- it may offer a more intuitive way for some users to configure the excluded paths directly in the `nuxt.config.ts`

### Additional information

Originally proposed here: https://github.com/sidebase/nuxt-auth/issues/81#issuecomment-1339004288

Interface proposal for config:
```ts
enableGlobalAppMiddleware: boolean | {
  enabled: boolean
  ignoresPaths: string[]
}
```

**Comment by BracketJohn** [comment]
wrongly closed via #90 - it did not contain these changes.

**Comment by BracketJohn** [comment]
closing this for now as so far no real demand was registered + implementation not planned. If you are interested in this feature feel free to comment or add a new issue (:
